server/channels/api4/status.go

```diff
@@ -78,7 +78,10 @@ func getUserStatusesByIds(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Write(js)
+	if _, err := w.Write(js); err != nil {
+		c.Logger.Warn("Error while writing response", mlog.Err(err))
+		return
+	}
```

Comment:
Nit pick: Let's keep the code consistent with the rest of the code base
```suggestion
	if _, err := w.Write(js); err != nil {
		c.Logger.Warn("Error while writing response", mlog.Err(err))
	}
```

Reply:
Sometimes I really don't know what I'm doing 😓

Sorry for this...

Reply:
No need to be sorry. Mistakes happen. To all of us.